packages/credential-showcase-openapi/package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "tsc --watch",
     "build": "tsc",
     "build:clean": "tsc --build --clean && tsc --build",
-    "generate:models": "mvn clean install"
+    "generate-models": "rimraf ./src && mvn clean generate-sources"
```

Comment:
Why are we changing from ':' to '-' ?

Reply:
please, keep as `:` for consistence with all the other packages with the repo

Reply:
yeah this came from my old fixes/generate-models branch.
It's fixed now